src/main/java/uk/gov/pay/api/resources/directdebit/DirectDebitPaymentsResource.java

```diff
@@ -67,4 +80,43 @@ public Response createPayment(@ApiParam(value = "accountId", hidden = true) @Aut
         return Response.created(publicApiUriGenerator.getDirectDebitPaymentURI(payment.getPaymentId()))
                 .entity(payment).build();
     }
+
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = DirectDebitSearchResponse.class),
+            @ApiResponse(code = 401, message = "Credentials are required to access this resource"),
+            @ApiResponse(code = 422, message = "Invalid parameter: from_date. See Public API documentation for the correct data formats", response = PaymentError.class),
```

Comment:
I think it's better to say `Invalid parameter` rather than `Invalid paramter: from_date` as that suggests the only parameter that can be invalid is the from_date. Think it's better to either specify no fields or all the fields.

Reply:
Yeh good idea

Reply:
hmm everywhere else we do provide an example field in the example error message.

Reply:
I think we should do what's sensible rather than copying code just because...

Reply:
I've updated it